acceptance/acceptance_test.go

```diff
@@ -178,6 +187,86 @@ func thereShouldBeNoWarningsInTheResult(ctx context.Context) error {
 	return nil
 }
 
+func opaEval(ctx context.Context, evalString string) (context.Context, error) {
+	ts, err := getTestState(ctx)
+	if err != nil {
+		return ctx, fmt.Errorf("opaEval get test state: %w", err)
+	}
+
+	cmd := exec.Command(
+		"go",
+		"run",
+		"github.com/enterprise-contract/ec-cli",
+		"opa",
+		"eval",
+		"--data",
+		"./policy", // everything
+		"--input",
+		ts.inputFileName,
+		evalString,
+	)
+	cmd.Dir = ts.variables["GITROOT"]
+
+	var stdout bytes.Buffer
+	var stderr bytes.Buffer
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
+
+	if err := cmd.Run(); err != nil {
+		return ctx, fmt.Errorf("running ec opa eval: %w\n%s", err, stderr.String())
+	}
+
+	ts.stdout = string(stdout.Bytes())
+
+	return setTestState(ctx, ts), nil
+}
+
+func theResultShouldBe(ctx context.Context, expectedResult string) error {
```

Comment:
There are some assumptions here that the result is in JSON format, and it is from `opa eval`. I think we can remove the assumption about `opa eval` without any consequence, see my other comment further down. Maybe call this `theJSONResultShouldBe` or `theOPAEvalResultShouldBe` depending on the remaining assumptions.